db-migrations/13_add_survey_mode_column.down.sql

```diff
@@ -0,0 +1,2 @@
+-- Individual ENUMs can not be dropped due to table validation
+DROP TYPE survey.survey_mode CASCADE;
```

Comment:
This isn't the best, you can't just get rid of one enum so a down script is questionable, this matches past patterns but I am not convinced